sharedTest/src/main/java/org/listenbrainz/sharedtest/utils/ResourceString.kt

```diff
@@ -38,6 +41,30 @@ object ResourceString {
         EntityTestUtils.loadResourceAsString("user_does_not_exist_error.json")
     }
     
+    val my_feed_page_1 by lazy {
+        EntityTestUtils.loadResourceAsString("my_feed_page_1.json")
```

Comment:
Can we make the mock files more descriptive and not keep them in the _number format?

Reply:
This is as descriptive as it gets since these are results of one endpoint but with different parameters (here, max_ts).